### ISSUE TITLE
To be implemented: Restoring S3 objects from Glacier

### ISSUE THREAD

**Issue by robvium** [spulec/moto#885]
# When calling object.restore_request

I get this 

```Unknown parameter in RestoreRequest: "GlacierJobParameters", must be one of: Days```

A call to restore_requests requires the key 'GlacierJobParameters' as of boto3 '1.4.4'

## How I'm calling the restore request.  (works on real data and buckets)

```
            restore_request = {
                'Days': self.warmth_duration_days,
                'GlacierJobParameters': {
                    'Tier': self.TIER.value
                }
            }
            response = object.restore_object(RestoreRequest=restore_request)
```

## Effect
moto can't be used for mocking restore requests with current version of boto3

**Comment by spulec** [comment]
Can you give some more code around what you are trying to achieve?

Based on my reading of the code, I don't believe we support Glacier restoring yet.

**Comment by bblommers** [comment]
Going to close this, as the `restore_object`-method has been implemented as of (at least) 2.0.0 - see https://github.com/spulec/moto/pull/3318/files.

Feel free to reopen if I'm missing anything, or open a new issue for any other missing features.
